script.lazytv/addon.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<addon id="script.lazytv"
+       name="LazyTV"
+       version="3.0.0"
+       provider-name="KodeKarnage">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+<extension point="xbmc.python.script"
+            library="default.py">
+            <provides>executable</provides>
+    </extension>
+
+    <extension point="xbmc.service" library="service.py"></extension>
+
+    <extension point="xbmc.addon.metadata">
+        <assets>
+            <icon>icon.png</icon>
+            <fanart>fanart.jpg</fanart>
+        </assets>
+        <summary lang="af">LazyTV</summary>
```

Comment:
would be nice to change all these lang attributes to the new default values: see https://github.com/xbmc/xbmc/tree/master/addons/skin.estuary/language